### ISSUE TITLE
This facial feature extraction network appears to use cross-entropy loss function and an Adam optimizer

### ISSUE THREAD

**Issue by tianmaojingling78** [anlanyu66/Federated_face_recognition#1]
Why do we need to use the cross-entropy loss function to train a face feature extraction network, so that the performance of the network will be improved with each iteration of the training? Isn't the triplet loss function generally used?
